src/Database/InMemoryProvider.js

```diff
@@ -73,7 +73,7 @@ class InMemoryProvider extends ADBProvider {
         const guild = await this.fetchGuild(gID);
         guild[key] = value;
         this.saveGuildSchema(gID, guild);
```

Comment:
This shouldn't call `this.saveGuildSchema` but should directly udpate the cache 
```suggestion
        this.axon.guildConfigs.set(gID, guild);
```
Ref: #53 that will change `saveGuildSchema` implementation to the correct one